firefox-ios/Client/Frontend/Home/UnifiedAds/UnifiedAdsProvider.swift

```diff
@@ -15,19 +16,20 @@ protocol UnifiedAdsProviderInterface: Sendable {
     /// - Parameters:
     ///   - timestamp: The timestamp to retrieve from cache, useful for tests. Default is Date.now()
     ///   - completion: Returns an array of Tiles, can be empty
-    func fetchTiles(timestamp: Timestamp, completion: @escaping @Sendable (UnifiedTileResult) -> Void)
+    func fetchTiles(timestamp: Shared.Timestamp, completion: @escaping @Sendable (UnifiedTileResult) -> Void)
```

Comment:
nit; It's not necessary to add the package on `Timestamp`. Same comment to lines 23 and 32.
```suggestion
    func fetchTiles(timestamp: Timestamp, completion: @escaping @Sendable (UnifiedTileResult) -> Void)
```